Source/GUI/BuildWindow.cs

```diff
@@ -809,9 +809,10 @@ void OnFlagSelected (FlagBrowser.FlagEntry selected)
 		private void craftSelectComplete (string filename,
 										  CraftBrowserDialog.LoadType lt)
 		{
-			control.LoadCraft (filename, flagURL);
 			control.craftType = craftlist.craftType;
 			craftlist = null;
+
+			control.LoadCraft(filename, flagURL);
```

Comment:
Does this change have an actual purpose? I don't see any issue with it either way, just curious.

Reply:
I've moved `craftlist = null;` before executing potentially crashing `control.LoadCraft` just to make sure the inner state of the GUI after the crash is at least in theory recoverable. There is a `GUI.enabled = craftlist == null;` in the SelectCraft, which, as I understand, will block the GUI in case of exception.

Overall, I've made all these "GUI recovery" adjustments only to avoid the need to reload the game every time I was loading craft that fails. Because it was not immediately apparent what's causing OrbitDriver's intervention/despawn and EL was fine to load some crafts and fail to load others, it took some non-trivial time to figure out that kOS is the culprit.

After that, I just thought it would be okay to keep this "recovery" adjustments, because from the user's point of view, empty GUI windows that stuck on screen and cannot be even closed - aren't very pleasant.

Reply:
Ah, ok. Suspenders-and-belt approach. No complaints here :)